js/templates/formError.html

```diff
@@ -0,0 +1,5 @@
+<ul class="unstyled errorList">
```

Comment:
The aforementioned `formError` template. The markup and css are most certainly tweakable.